config.md

```diff
@@ -7,304 +7,170 @@ This specification outlines the JSON format describing images for use with a con
+     **NOTE:** This JSON structure value is unusual because it is a direct JSON serialization of the Go type <code>map[string]struct{}</code> and is represented in JSON as an object mapping its keys to an empty object.
+
+   - **Env** *array of strings*, OPTIONAL
+
+     Entries are in the format of <code>VARNAME="var value"</code>.
+     These values act as defaults and are merged with any specified when creating a container.
+
+   - **Entrypoint** *array of strings*
+
+     A list of arguments to use as the command to execute when the container starts.
+     This value acts as a  default and is replaced by an entrypoint specified when creating a container. This field MAY be "null".
+
+   - **Cmd** *array of strings*, OPTIONAL
+
+     Default arguments to the entrypoint of the container.
+     These values act as defaults and are replaced with any specified when creating a container.
+     If an `Entrypoint` value is not specified, then the first entry of the `Cmd` array should be interpreted as the executable to run.
+
+   - **Volumes** *object*, OPTIONAL
+     A set of directories which should be created as data volumes in a container running this image. This field MAY be "null".
+     If a file or folder exists within the image with the same path as a data volume, that file or folder is replaced with the data volume and is never merged. **NOTE:** This JSON structure value is unusual because it is a direct JSON serialization of the Go type <code>map[string]struct{}</code> and is represented in JSON as an object mapping its keys to an empty object.
+
+   - **WorkingDir** *string*, REQUIRED
+
+     Sets the current working directory of the entrypoint process in the container.
+     This value acts as a default and is replaced by a working directory specified when creating a container.
+
+- **rootfs** *object, REQUIRED
+
+   The rootfs key references the layer content addresses used by the image.
+   This makes the image config hash depend on the filesystem hash.
+
+  - **type** *string*, REQUIRED
+
+     MUST be set to `layers`.
+     Implementations MUST generate an error if they encounter a unknown value while verifying or unpacking an image.
+
+  - **diff_ids** *array*, REQUIRED
+
+     An array of layer content hashes (`DiffIDs`), in order from bottom-most to top-most.
+
+- **history** *array of object*
+
+ Describes the history of each layer.
+ The array is ordered from bottom-most layer to top-most layer.
+ The object has the following fields:
+
+  - **created** *string*, OPTIONAL
+
+     Creation time, expressed as a ISO-8601 formatted combined date and time
+
+  - **author** *string*, OPTIONAL
+
+     The author of the build point.
+
+  - **created_by** *string*, OPTIONAL
+
+     The command which created the layer.
+
+  - **comment** *string*, OPTIONAL
+
+     A custom message set when creating the layer.
+
+  - **empty_layer** *string*
+
+     This field is used to mark if the history item created a filesystem diff, OPTIONAL
```

Comment:
“OPTIONAL“ should go on the line with **empty_layer**, like you do with all the other properties.